nats/src/main/java/io/micronaut/nats/reactive/RxJavaReactivePublisher.java

```diff
@@ -44,24 +43,23 @@ public RxJavaReactivePublisher(@Parameter Connection connection) {
 
     @Override
     public Publisher<Void> publish(PublishState publishState) {
-        return getConnection().flatMapCompletable(con -> publishInternal(publishState, con)).toFlowable();
+        return getConnection().flatMap(con -> publishInternal(publishState, con));
     }
```

Comment:
Please rename this class to `ReactorReactivePublisher`

Reply:
Done 😄